draft/spec/index.html

```diff
@@ -854,7 +854,17 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="extensions">
+        <h2>Extensions directory</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
+            of extending the functionality of an OCFL Object. Sub-directories of the extensions directory MUST be named
```

Comment:
* Here we do not make it illegal to have files in the extension directory, I think that is the intention
 * I would also say the sub-directories SHOULD be named according to a registered name rather then MUST

Reply:
SHOULD would leave open the possibility of multiple things implementing the same functionality but with different local names. This would make it difficult for clients to understand that it can operate on that content. (The alternative would be to use a NamAsTe file to declare an extension's identity in the root directory of that extension, e.g., (`extensions/versioning/0=mutable_head-1.0`), which would allow us to name things differently. But this was an unpopular suggestion on the editorial call).

Reply:
Aren't the directories under <code>extensions</code> a namespace that references OCFL extentions - in that case, there should only be a single instance of any one type.

Reply:
My concern is that by making it a SHOULD we leave open the possibility that people name their directories in `extensions` in a way that diverges from the name that is registered by the extension.

Reply:
@ahankinson I'm agreeing with you!